ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/dcat/dcatserver/services/DcatServerServiceImpl.java

```diff
@@ -51,7 +51,7 @@ public Model getComposedDcat() {
 				.findSingleDcatServer()
 				.ifPresent(dcatServer -> composedDcat.add(getStatementsForComposedDcat(dcatServer)));
 
-		dcatShaclValidator.validate(composedDcat, null);
+		dcatShaclValidator.validate(composedDcat);
```

Comment:
Now you are calling the impl from the abstract class directly and bypassing the if check from the specific implementation:

```
        if (isValidatorActive()) {
            Model model = (Model) target;
            validate(model);
        }
```